Translations/pt_BR.po

```diff
@@ -1196,17 +1187,19 @@ msgstr "Forçar o modo Diablo mesmo que o arquivo hellfire.mpq seja encontrado"
 #. TRANSLATORS: Commandline Option
 #: Source/diablo.cpp:903
 msgid "Use alternate nest palette"
-msgstr ""
+msgstr "Usar paleta de ninho alternativa"
```

Comment:
Ninho......... oof, this is cringey in portuguese. Don't know a more proper translation though. However, the sentence is still wrong:
```suggestion
msgstr "Usar paleta alternativa do ninho"
```